pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -46,6 +46,13 @@
     @Parameter(names = { "--auth-params" }, description = "Authentication parameters, e.g., \"key1:val1,key2:val2\".")
     String authParams = null;
 
+    @Parameter(names = { "--tls-allow-insecure" }, description = "Allow TLS insecure connection")
+    Boolean tlsAllowInsecureConnection;
+
+
+    @Parameter(names = { "--tls-enable-hostname-verification" }, description = "Enalbe TLS common name verification")
+    Boolean tlsEnableHostnameVerification;
```

Comment:
Shouldn't it be true by default

Reply:
yes, probably, but it should be consistent with the same setting on client library, which is currently default to false